### PR TITLE
Implement disposition field in SQL backend

### LINKED ISSUE #3447: [FEATURE]: Add Disposition parameter in SqlBackend creation
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem statement

When running the `databricks labs ucx export-assessment`, some customer are facing the following issue:

> BadRequest: Inline byte limit exceeded. Statements executed with disposition=INLINE can have a result size of at most 26214400 bytes. Please execute the statement with disposition=EXTERNAL_LINKS if you want to download the full result.

This configuration can also introduce limits in other executions in which SQL statement results are not negligible. 

### Proposed Solution

Requiring the `Disposition` value during the UCX installation can allow users to select the proper disposition method to use when running large queries. This parameter can be added then in the `config.yml` file and used for the `SqlBackend` definition. Using this approach, large queries will not fail.

### Additional Context

I can work on it and introduce this feature if it seems compliant with UCX framework.

### PR DESCRIPTION
**Pull request by micheledaddetta-databricks**
### Description

This feature aims to the chance of running SQL statement for assessment results export in case of large workspaces with a large amount of findings.  
Introducing the `query_statement_disposition` value during the UCX installation can allow users to select the proper disposition method to use when running large queries. This parameter is added in the `config.yml` file and used for the SqlBackend definition. Using this approach, large queries will not fail.

### Changes
Added SQL query statement disposition choice during the UCX installation
Added `query_statement_disposition` value in the config file

### Linked issues

Resolves #3447

### Functionality

- [X] modified existing commands: `databricks labs install ucx`, `databricks labs ucx export-assessment`

### Tests

- [X] manually tested

### CODE REVIEW

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
@micheledaddetta-databricks : Thanks for the contribution. We need @gueniai to confirm your Github handle has the CLA signed. Can you make sure your git commits signature is verified (https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)?

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -253,6 +253,14 @@
     "audioread.rawread": [],
```
[the remark this answers is not in the document]
Comment (micheledaddetta-databricks) [reply src/databricks/labs/ucx/source_code/known.json]: Done

File: src/databricks/labs/ucx/install.py [hunk 2]
```diff
@@ -259,6 +260,12 @@ def _prompt_for_new_installation(self) -> WorkspaceConfig:
         recon_tolerance_percent = int(
             self.prompts.question("Reconciliation threshold, in percentage", default="5", valid_number=True)
         )
+
+        query_statement_disposition = self.prompts.confirm(
+            "Do you want to use the EXTERNAL_LINKS disposition value for query statements? (Only needed with large workspaces)"
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/install.py:265]:
```suggestion
            "Do you want to use the `EXTERNAL_LINKS` disposition for query statements? (Only needed when exporting more than 25 MiB of data from workspaces with many resources)"
```

**Comment by micheledaddetta-databricks** [comment]
Hello @JCZuurmond thanks for taking in charge the request! I just set up my git commits signature. Is that an issue for previous commits?

**Comment by JCZuurmond** [comment]
> Hello @JCZuurmond thanks for taking in charge the request! I just set up my git commits signature. Is that an issue for previous commits?

I cannot tell, @gueniai can maybe figure this out for us

**Comment by gueniai** [comment]
Hi @micheledaddetta-databricks , thank you for the PR and for verifying the signature. Unfortunately the previous commits are not automatically also verified. You can try with an interactive rebase, but I have a feeling you're going to need to re-create the PR :/

**Pull request closed without merging**